src/web/app/pages-admin/admin-search-page/admin-search-page.component.html

```diff
@@ -36,16 +36,28 @@
       <tbody>
         <ng-container *ngFor="let instructor of instructors; let i = index">
           <tr (click)="instructor.showLinks = !instructor.showLinks">
-            <td>{{ instructor.courseId }}</td> <!-- TODO: tooltip course name -->
+            <td>
+              {{ instructor.courseId }}
+              <br/>
+              <div class="col-sm-1">
+                <button *ngIf="!instructor.showLinks"
+                        class="btn"><i class="fas fa-chevron-circle-down"
+                                       style="color: blue;"></i></button>
```

Comment:
I normally don't pick on indentations, but this is very weird.
Why not just do a simple one:
```html
<button *ngIf="!instructor.showLinks" class="btn">
  <i class="fas fa-chevron-circle-down" style="color: blue;"></i>
</button>
```

Reply:
This was the unfortunate victim of copy-paste :P